### PR TITLE
Torus connector important bug fixes

### PR DESCRIPTION
**Pull request by chaitanyapotti**
The following changes have been made in this PR:
- Update torus version
- set torus to undefined on deactivate to fix issue with relogin
- Start using ethereum.request instead of ethereum.send(deprecated) for getAccount api

### CODE REVIEW

**Comment by chaitanyapotti** [comment]
@NoahZinsmeister Can we get this merged asap.?

**Review by estevanpedro — APPROVED** [review]

File: packages/torus-connector/src/index.ts [hunk 1]
```diff
@@ -46,13 +46,14 @@ export class TorusConnector extends AbstractConnector {
...
   public async getAccount(): Promise<null | string> {
-    return this.torus.ethereum.send('eth_accounts').then((accounts: string[]): string => accounts[0])
+    return this.torus.ethereum.request({ method: 'eth_accounts' }).then((accounts: string[]): string => accounts[0])
   }
 
   public async deactivate() {}
 
   public async close() {
     await this.torus.cleanUp()
+    this.torus = undefined
```
Comment (h0tw4t3r) [inline packages/torus-connector/src/index.ts:56]: similar approach is really needed here https://github.com/NoahZinsmeister/web3-react/blob/v6/packages/walletconnect-connector/src/index.ts#L110

**Review by h0tw4t3r — APPROVED** [review]

**Comment by chaitanyapotti** [comment]
@NoahZinsmeister A lot of people are waiting on this PR. 
can you please merge this?

**Comment by sakulstra** [comment]
@NoahZinsmeister is there anything blocking a merge?
This is starting to cause problems as the currently bundled @toruslabs packages throw unsupported engine errors on node lts

**Comment by NoahZinsmeister** [comment]
sorry was having some trouble working with this PR locally, closing in favor of #354, which does the same exact thing

**Comment by h0tw4t3r** [comment]
> similar approach is really needed here https://github.com/NoahZinsmeister/web3-react/blob/v6/packages/walletconnect-connector/src/index.ts#L110
@NoahZinsmeister, does WalletConnect also needs this fix?

**Pull request closed without merging**